### PR TITLE
 Add `List[Expr[T]]` to `Expr[List[T]]` conversion in the library

### PR DESCRIPTION
**Pull request by nicolasstucki**
Both `scalatest` and the interpolators need this operation and they are probably not the only ones.

### CODE REVIEW

File: library/src-scala3/scala/quoted/package.scala [hunk 1]
```diff
@@ -0,0 +1,18 @@
+package scala
+
+package object quoted {
+
+  implicit class LiftExprOps[T](val x: T) extends AnyVal {
+    def toExpr(implicit ev: Liftable[T]): Expr[T] = ev.toExpr(x)
+  }
+
+  implicit class ListOfExprOps[T](val list: List[Expr[T]]) extends AnyVal {
```
Comment (allanrenucci) [inline library/src-scala3/scala/quoted/package.scala]: Don't you have this conversion for free if you make `Liftable[List[T]]` implicitly available?

Comment (nicolasstucki) [reply library/src-scala3/scala/quoted/package.scala]: No, `Liftable[List[T]]` transforms `List[T]` into `Expr[List[T]]`. But here we transform `List[Expr[T]]` into `Expr[List[T]]`.

Comment (nicolasstucki) [reply library/src-scala3/scala/quoted/package.scala]: This is a generalisation of your [`liftListOfAny`](
https://github.com/lampepfl/dotty/pull/5543/files#diff-c03a5fbe7d1df314004a2fceb5b6df73L59).

**Comment by nicolasstucki** [comment]
This PR may silently conflict with #5520

**Comment by nicolasstucki** [comment]
`Seq` should probably be implemented in a library.

**Pull request merged**